docs/cmdline-opts/page-header

```diff
@@ -99,6 +99,15 @@ getting many files from the same server will not do multiple connects /
 handshakes. This improves speed. Of course this is only done on files
 specified on a single command line and cannot be used between separate curl
 invokes.
+.SH OUTPUT
+If not told otherwise, curl writes output to stdout. It can be instructed to
```

Comment:
This is wrong. Verbose and errors are written to stderr

Reply:
In my view that's not "output" but from your reaction I understand I should probably qualify that in the text...

Reply:
```
curl --help category | grep verbose
 verbose     Options related to any kind of command line output of curl
```
It is output :P

Reply:
Again, that's not matching my personal terminology. Verbose output is not plain "output".

Reply:
To me, the output of curl is the data it outputs.

Reply:
Ok your decision then. In my personal opinion output means everything visible on the shell. Of course this isn't standardized somehow but every average shell displays stdout and stderr.